tests/accessinvite_test.go

```diff
@@ -0,0 +1,51 @@
+package tests
+
+import (
+	"fmt"
+	accessservices "github.com/jfrog/jfrog-client-go/access/services"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccessInvite(t *testing.T) {
+	initAccessTest(t)
+	t.Run("invite", testInviteUser)
+}
+
+func testInviteUser(t *testing.T) {
+	randomMail := fmt.Sprintf("test%s@jfrog.com", timestampStr)
+	UserParams := getTestInvitedUserParams(randomMail)
+	err := testUserService.CreateUser(UserParams)
+	assert.NoError(t, err)
+
+	// Second invitation should fail because we can invite user only once a day (Access's internal reasons).
```

Comment:
Allowing to invite the specific user only once a day doesn't make sense. Limiting this to 5 time a day make more sense.